crypto3/libs/algebra/include/nil/crypto3/algebra/curves/detail/forms/twisted_edwards/extended_with_a_minus_1/element_g1.hpp

```diff
@@ -221,21 +221,18 @@ namespace nil {
 
                         /*************************  Arithmetic operations  ***********************************/
 
-                        constexpr curve_element operator=(const curve_element &other) {
-                            // handle special cases having to do with O
+                        constexpr curve_element& operator=(curve_element<params_type, form, curves::coordinates::affine> const &other) {
                             this->X = other.X;
                             this->Y = other.Y;
-                            this->T = other.T;
```

Comment:
Why do we need this change. At least add a comment, so the next person will read this easier.

Reply:
Original `operator=` is removed, as it has trivial implementation. This is `operator=` from a different class - affine coordinates.